src/com/company/Contact.java

```diff
@@ -0,0 +1,23 @@
+package com.company;
+
+public class Contact {
+    String name;
+    String telephone;
+    String email;
+
+    Contact(String nameContact, String telephoneContact, String emailContact){
+        name=nameContact;
+        telephone=telephoneContact;
+        email=emailContact;
+    }
+
+    public String giveName(){
+        return name;
+    }
+    public String giveTelephone(){
+        return telephone;
+    }
+    public String giveEmail(){
+        return email;
```

Comment:
@SFDC-Melby-Arguedas When your are refering to class attribute, do it with **this** keyword